crates/mysten-network/src/multiaddr.rs

```diff
@@ -23,7 +23,7 @@ impl Multiaddr {
         Self(inner)
     }
 
-    pub(crate) fn iter(&self) -> ::multiaddr::Iter<'_> {
+    pub fn iter(&self) -> ::multiaddr::Iter<'_> {
```

Comment:
@bmwill this was the easiest way for me to parse out the parts of a Multiaddr, if you think that sucks I can figure out another way